charts/argocd-config/templates/app-of-apps.yaml

```diff
@@ -7,13 +7,18 @@ spec:
     server: https://kubernetes.default.svc
     namespace: argocd
   project: {{ .Values.environment }}
-  source:
-    repoURL: {{ .Values.repoUrls.wbstack }}
-    targetRevision: 1.0.5
+  sources:
+    - repoURL: {{ .Values.repoUrls.wbstack }}
+    targetRevision: 1.0.6
     chart: argocd-apps
     helm:
       values: |
 {{ toYaml .Values | indent 8 }}
+      valueFiles:
```

Comment:
does this work? I vaguely remember that only `values` or `valuesFiles` will get used, but I could be wrong

Reply:
Unlesss I've ended up in a right muddle yes, this seems to work fine